system_strings/sysmes_text.en

```diff
@@ -2229,83 +2229,118 @@ Cleaning up the aftermath
 }
 [sha:ffb72a762eb8497d77f5cf2674a3e1feff6fbc94]{
 -- 六日目・朝。
+// @flowchart_title
 6th day ― Morning
 }
 [sha:1693642317818b0d076ab773239f161cc356c24a]{
 -- 一切の望みを捨てよ。
-Abandoning all hope.
+// @flowchart_title
+Abandoning All Hope
 }
 [sha:65a08d7f151f2845e411eed735cc6454e300e569]{
 -- 素直に謝る。
-I apologize sincerely.
+// Reference 42624
+// @flowchart_title
+I apologize sincerely
```

Comment:
Just realized,
while it's important to reference the offset here, it's equally important to note in the script that the line in question is related to the title
Just so that if someone changes it, they also know to look at the titles